1-mandatory/1-alarmclock/alarmclock.js

```diff
@@ -1,9 +1,42 @@
-function setAlarm() {}
+function setAlarm() {
+  let inputVal = document.getElementById("alarmSet").value;
+  let titleEl = document.querySelector("#timeRemaining");
+  let time = splitTime(inputVal);
+  titleEl.innerText = `Time Remaining: ${formatVal(time.minute)}:${formatVal(
+    time.second
+  )}`;
+
+  let refreshIntervalId = setInterval(() => {
+    if (inputVal > 0) {
+      inputVal--;
+      time = splitTime(inputVal);
+      titleEl.innerText = `Time Remaining: ${formatVal(
+        time.minute
+      )}:${formatVal(time.second)}`;
+    }
+    if (inputVal === 0) {
+      inputVal = null;
+      playAlarm();
+      clearInterval(refreshIntervalId);
+    }
+  }, 1000);
+}
```

Comment:
Your alarm functionality works really well, nice job. I also really like that you've thought about your variable names to make your code easy to follow.
A nice little thing to add might be to clear the input once the alarm has been stopped :)